Makefile

```diff
@@ -210,20 +214,23 @@ endif
 $(HOME)/go/bin/mockery:
 	go get github.com/vektra/mockery/.../
 
+dist/update-mocks: $(HOME)/go/bin/mockery $(MOCK_FILES)
```

Comment:
de-couple the mocks from other codegen, ultimately I'd like codegen to be smart enough to only run if needed

Reply:
Nice!